tgui/src/interfaces/shuttle_manipulator/status.ract

```diff
@@ -1,14 +1,33 @@
+<table width="100%">
 {{#each data.shuttles}}
-  <ui-section label='{{name}} ({{id}})'>
```

Comment:
https://github.com/tgstation/tgstation/pull/39870 This would be nice to port also